src/software/ai/hl/stp/tactic/tactic_visitor.h

```diff
@@ -46,31 +39,22 @@ class TacticVisitor
      *
      * @param tactic The tactic to visit
      */
-
-    virtual void visit(const ChipTactic &tactic)                 = 0;
     virtual void visit(const CreaseDefenderTactic &tactic)       = 0;
-    virtual void visit(const DefenseShadowEnemyTactic &tactic)   = 0;
-    virtual void visit(const DribbleTactic &tactic)              = 0;
-    virtual void visit(const GetBehindBallTactic &tactic)        = 0;
     virtual void visit(const GoalieTactic &tactic)               = 0;
-    virtual void visit(const GoalieTestTactic &tactic)           = 0;
-    virtual void visit(const KickTactic &tactic)                 = 0;
-    virtual void visit(const KickoffChipTactic &tactic)          = 0;
     virtual void visit(const MoveTactic &tactic)                 = 0;
-    virtual void visit(const MoveTestTactic &tactic)             = 0;
     virtual void visit(const AttackerTactic &tactic)             = 0;
     virtual void visit(const PassDefenderTactic &tactic)         = 0;
     virtual void visit(const PenaltyKickTactic &tactic)          = 0;
     virtual void visit(const PenaltySetupTactic &tactic)         = 0;
-    virtual void visit(const PivotKickTactic &tactic)            = 0;
     virtual void visit(const ReceiverTactic &tactic)             = 0;
     virtual void visit(const ShadowEnemyTactic &tactic)          = 0;
     virtual void visit(const StopTactic &tactic)                 = 0;
-    virtual void visit(const StopTestTactic &tactic)             = 0;
     virtual void visit(const MoveGoalieToGoalLineTactic &tactic) = 0;
     virtual void visit(const PrepareKickoffMoveTactic &tactic)   = 0;
-    virtual void visit(const PlaceBallTactic &tactic)            = 0;
     virtual void visit(const PlaceBallMoveTactic &tactic)        = 0;
-    virtual void visit(const WallKickoffTactic &tactic)          = 0;
     virtual void visit(const AvoidInterferenceTactic &tactic)    = 0;
+    virtual void visit(const SkillTactic &tactic)                = 0;
```

Comment:
I think we may want to delete this. We'd like every tactic to define all the visitors